src/main/kotlin/lotto/LottoSeller.kt

```diff
@@ -0,0 +1,15 @@
+package lotto
+
+class LottoSeller(
+    private val lottoFactory: LottoFactory
+) {
+    fun sell(totalPurchasePrice: Int): Lottos {
+        require(totalPurchasePrice > 0) { "총 구매금액은 0 보다 커야합니다" }
+        return Lottos(List(totalPurchasePrice / LOTTO_PRICE) { lottoFactory.create(1000) })
```

Comment:
상수로 정의한 `LOTTO PRICE` 를 사용해도 될 것 같네요! 
```suggestion
        return Lottos(List(totalPurchasePrice / LOTTO_PRICE) { lottoFactory.create(LOTTO_PRICE) })
```